src/Nethermind/Nethermind.Runner/Ethereum/JsonRpcRunner.cs

```diff
@@ -76,6 +76,7 @@ public async Task Start(CancellationToken cancellationToken)
                     s.AddSingleton(_jsonRpcUrlCollection);
                     s.AddSingleton(_webSocketsManager);
                     s.AddSingleton(_rpcAuthentication);
+                    s.AddSingleton(_api);
```

Comment:
I don't like this, we want to remove/hide api not inject it

Reply:
Changed to inject them individually